netjsonconfig/schema.py

```diff
@@ -491,10 +511,11 @@
             }
         },
         "encryption_wpa_enterprise_ap": {
-            "title": "WPA2/WPA Enterprise (access point)",
+            "title": "WPA3/WPA2/WPA Enterprise (access point)",
```

Comment:
similar concept here as above, that is, I think we have to create a new dedicated section for WPA3 enterprise which makes use of `encryption_mfp_property_required`